examples/doc/example.md

```diff
@@ -52,7 +52,7 @@ Represents the status of a vehicle booking.
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
 | id | [int32](#int32) | required | Unique booking status ID. |
-| description | [string](#string) | required | Booking status description. E.g. &quot;Active&quot;. |
+| description | [string](#string) | required | Booking status description. E.g. "Active". |
```

Comment:
All changes to this file are unrelated. Not sure why this changed when I ran `make all` from the _examples_ directory. Was it out of date?

If you'd prefer I can pull this change out